regional_mom6/regional_mom6.py

```diff
@@ -3310,8 +3494,15 @@ def regrid_tides(
         self.encode_tidal_files_and_output(ds_ap, "tz")
 
         ########### Regrid Tidal Velocity ######################
-        regrid_u = rgd.create_regridder(tpxo_u[["lon", "lat", "uRe"]], coords, ".temp")
-        regrid_v = rgd.create_regridder(tpxo_v[["lon", "lat", "vRe"]], coords, ".temp2")
+
+        # Change to t-point coords
+        if rotational_method == rot.RotationMethod.KEITH_DOUBLE_REGRIDDING:
```

Comment:
I wonder if it's worth keeping KEITH_DOUBLE_REGRIDDING in the code. Unless there is a use case of this method over Fred's method, I suggest we remove it to keep the code simpler.

Reply:
Removing KEITH_DOUBLE_REGRIDDING!